src/node.cc

```diff
@@ -4531,6 +4531,8 @@ inline int Start(Isolate* isolate, IsolateData* isolate_data,
     } while (more == true);
   }
 
+  AsyncWrap::RunDestroyCbs(&env);
```

Comment:
@addaleax ah. think I see what's going on. will get another fix for this in a moment.